reddit_liveupdate/utils.py

```diff
@@ -10,30 +10,23 @@
 
 def pretty_time(dt, allow_relative=True):
     ago = datetime.datetime.now(pytz.UTC) - dt
+    date_format = ''
```

Comment:
Just as a btw this line isn't needed since `date_format` is being set in all branches of the if statement